src/test/java/Homework18mc.java

```diff
@@ -0,0 +1,15 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18mc extends BaseTest {
+
+    @Test(priority = 1, description = "Play a song and assert a song is playing")
+    public void playSong() {
+        login("demo@class.com", "te$t$tudent");
+        playSong();
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/216758493-3fbedd73-8b4f-466f-97a9-82c1c44c79d0.png)